### PR TITLE
Temporarily disabling macOS build

### PR DESCRIPTION
**Pull request by DXCanas**
### Summary

Mac builds have been timingout at the upload step as of late. Disabling until tomorrow, when I can get to the build machine for more debugging.

### Reviewer guidance

See if the mac build has stopped uploading without breaking the pipeline

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by DXCanas** [comment]
@lyw07 I tagged Devon, but if you've got time to review, I'd appreciate it.

Basically the Mac randomly timeouts. I'm going to be in office tomorrow to give it a hardline connection to a router so that we don't have to deal with WiFi issues.

File: .buildkite/setup_and_upload_artifact.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ mkdir -p dist
 buildkite-agent artifact download 'dist/*.pex' dist/
 buildkite-agent artifact download 'dist/*.whl' dist/
 buildkite-agent artifact download 'dist/*.tar.gz' dist/
-buildkite-agent artifact download 'dist/*.deb' dist/
+# buildkite-agent artifact download 'dist/*.deb' dist/
```
Comment (lyw07) [inline .buildkite/setup_and_upload_artifact.sh]: I guess you were trying to comment out .dmg?

**Review by lyw07 — CHANGES_REQUESTED** [review]

File: .buildkite/setup_and_upload_artifact.sh (hunk 1 above)
Comment (DXCanas) [reply .buildkite/setup_and_upload_artifact.sh]: 🤦‍♂

Comment (DXCanas) [reply .buildkite/setup_and_upload_artifact.sh]: Good catch

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -31,7 +31,11 @@ steps:
 
   - trigger: "kolibri-raspbian-image"
     label: ":raspberry-pi:"
-    build: *triggered_build_attributes
+    build: &triggered_build_attributes
```
Comment (lyw07) [inline .buildkite/pipeline.yml:34]: sorry, just curious - what's the difference between `*` and `&`?

Comment (DXCanas) [reply .buildkite/pipeline.yml:34]: It's kind of (barely) like pointer syntax.

Before, the MacOS object was defining a [YAML anchor](https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html) with build properties to be used by all the trigger steps - `triggered_build_attributes`

But I commented it out, so the definition vanished. Now the definition lives in the `kolibri-raspbian-image` object

Comment (lyw07) [reply .buildkite/pipeline.yml:34]: oh that's cool! thank you for the explanation 👍

**Review by lyw07 — APPROVED** [review]
Looks good to me!

**Comment by lyw07** [comment]
Hi @DXCanas Do you mind pulling from latest release branch so we can let the build pass before merging this in? Thank you! :)

**Pull request merged**